### ISSUE TITLE
Autocomplete inside navbar display issue

### ISSUE THREAD

**Issue by roemhildtg** [picturepan2/spectre#379]
Is there an issue with using the autocomplete component inside a navbar?

It seems to display weird: http://jsbin.com/yavevokire/edit?html,js,output

**Comment by picturepan2** [comment]
@roemhildtg Yes. It is a bug. I think I should move Autocomplete to the Experimentals section.

**Comment by Randore** [comment]
@picturepan2 why don't you use datalist?
http://html5doctor.com/html5-forms-introduction-and-new-attributes/
